flink-connectors/flink-connector-hive/src/main/java/org/apache/flink/table/catalog/hive/HiveDatabaseUtil.java

```diff
@@ -32,8 +33,6 @@
 import static org.apache.flink.sql.parser.hive.ddl.SqlAlterHiveDatabase.ALTER_DATABASE_OP;
 import static org.apache.flink.sql.parser.hive.ddl.SqlAlterHiveDatabaseOwner.DATABASE_OWNER_NAME;
 import static org.apache.flink.sql.parser.hive.ddl.SqlAlterHiveDatabaseOwner.DATABASE_OWNER_TYPE;
-import static org.apache.flink.table.catalog.hive.HiveCatalog.isGenericForCreate;
-import static org.apache.flink.table.catalog.hive.HiveCatalog.isGenericForGet;
 
 /** Util methods for processing databases in HiveCatalog. */
 public class HiveDatabaseUtil {
```

Comment:
I think this class shouldn't exist, all the methods should be private method in HiveCatalog